src/modules/escape-hatch/selectors/escape-hatch-data.js

```diff
@@ -71,7 +76,14 @@ export const getEscapeHatchData = createSelector(
       }
     })
 
-    data.fundsAvailableForWithdrawal = data.rep + data.eth > 0
+    Object.keys(orders).forEach((orderId) => {
+      const order = orders[orderId]
+      data.eth = data.eth.plus(createBigNumber(speedomatic.fix(order.tokensEscrowed)))
+      data.shares = data.shares.plus(createBigNumber(speedomatic.fix(order.sharesEscrowed)))
+      data.gas = data.gas.plus(CANCEL_ORDER_GAS_ESTIMATE)
+    })
+
+    data.fundsAvailableForWithdrawal = data.rep + data.eth + data.shares > 0
 
```

Comment:
is '+' same as .plus with BigNumbers?

`fundsAvailableForWithdrawal` seems to be used as a number but is a bool. 

referencing src/modules/escape-hatch/components/escape-hatch.jsx
`  const fundsAvailableForWithdrawal = nextProps.escapeHatchData.fundsAvailableForWithdrawal > 0`

How about using it as a bool or use it as a number. If use as number use .plus(....).plus(...).toNumber()

Reply:
ah yea good point, will do